src/pages/Verification/Verification.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState, useEffect } from "react";
+import { Typography, Container, Box, Button } from "@mui/material";
+import { MailOutline } from '@mui/icons-material';
+import { selectToken, selectUser } from "@/redux/userSlice";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { RootState } from '@/redux/store';
+import { BASE_URL } from '@/config/default';
+import axios, { AxiosError, AxiosResponse } from "axios";
+import AlertMessage from "@/components/ui/AlertMessage";
+
+interface AuthResponse {
+  success: boolean;
+  data?: any;
+  error?: string;
+  message?: string;
+}
+
+export default function Verification() {
+  const token = useSelector((state: RootState) => selectToken(state));
+  const user = useSelector((state: RootState) => selectUser(state));
+  const navigate = useNavigate();
+  const [countdown, setCountdown] = useState(120);
+  const [resendDisabled, setResendDisabled] = useState(true);  
+  const [alert, setAlert] = useState<{ message: string; severity: "error" | "warning" | "info" | "success" } | null>(null);
```

Comment:
It getting a bit crowded to read the `severity` object. Is there a way you can extract the `severity` type alone somewhere in this file?

Reply:
Wouldn't it be unnecessary to create other file just for the severity? Instead I used ```type Severity = "error" | "warning" | "info" | "success";```

Reply:
No need to create a separate file just define it in the same file somewhere in the beginning